packages/itwinui-css/src/button/base.scss

```diff
@@ -32,6 +32,18 @@ $disabledButtonSelectors: '[disabled], :disabled, [aria-disabled="true"], [data-
   &:is(#{$disabledButtonSelectors}) {
     --_iui-button-icon-fill: var(--iui-color-icon-disabled);
   }
+
+  &:is([data-iui-loading='true']) {
+    > :where(span) {
```

Comment:
This feels a little brittle. I'm thinking we should add a class for the button-label

Reply:
Yeah, adding a `.iui-button-label` class for the `<span>` elements inside of the button makes sense to me.

Reply:
> This feels a little brittle. I'm thinking we should add a class for the button-label

Yes, sounds good, since you're also adding `.iui-button-spinner`. But that can come in another PR if that was your plan.

Reply:
added in [`1c87486`](https://github.com/iTwin/iTwinUI/pull/2058/commits/1c87486e095e194280dcc6d18066f47978419c7a)